packages/Upgrade/src/api/controllers/validators/ScheduledJobsParamsValidator.ts

```diff
@@ -1,8 +1,7 @@
-import { IsNotEmpty, IsDefined, IsUUID } from 'class-validator';
+import { IsNotEmpty, IsDefined } from 'class-validator';
 
 export class ScheduledJobsParamsValidator {
   @IsNotEmpty()
-  @IsUUID()
```

Comment:
This is going to be an uuid right? Why have we removed this?

Reply:
Actually we have used expeirmentId_event-type (start experiment, end experiment) so it will not have isUUID